### DISCUSSION TITLE
Newbie Link Question

### DISCUSSION THREAD

**Discussion by RevRob1** [samschott/maestral#512] [Q&A]
Hello there - thanks for this app, Sam.

I've installed and linked the app to my work dropbox. Now trying to do the same with my personal dropbox, but not having much success. Any help gladly received. 
Many thanks,
Robert

**Comment by samschott** [comment]
> Now trying to do the same with my personal dropbox, but not having much success.

What did you try and how did it fail?

**Comment by RevRob1** [comment]
I tried to follow the intructions on the webpage using terminal, but it just comes back with zsh: no files found.

Apologies, I'm new to all of this!

<img width="771" alt="CleanShot 2021-11-04 at 15 47 45@2x" src="https://user-images.githubusercontent.com/93724054/140369720-6ab4d2c2-9cf1-4d87-8aef-8ab793318842.png">
t

**Reply by samschott** [reply]
Does it really say "zsh: no files found" or "zsh: command not found"? You will need to install the command line tool first (Preferences > Command Line Tool > Install). There are reports that this sometimes fails on M1 macs, since I don't have one of those I could not yet work on a fix...

If it succeeds, the easiest way to then link a new account would be to use the following command:
```shell
maestral gui -c new
```
This will create a config called "new" and open a new instance of Maestral with the familiar setup dialog.

**Reply by RevRob1** [reply]
All now working - thanks!

**Comment by RevRob1** [comment]
Thanks, Sam - so now I'm getting to the dialog box etc.

I get to the dropbox screen, which asks me which account I want to link to.

I click on the 'personal' account (the work one is already linked) and the image below is what I get.

Many thanks,  Robe
<img width="606" alt="CleanShot 2021-11-05 at 09 41 21@2x" src="https://user-images.githubusercontent.com/93724054/140491141-6086b0bc-859f-4849-9e8b-028efc0f87f8.png">
rt

**Reply by RevRob1** [reply]
BTW, I am signed into dropbox when I do this.

**Reply by samschott** [reply]
Could you try to sign out of the work account and into the personal account only?
